.github/workflows/ecli-image-aarch64.yml

```
@@ -0,0 +1,78 @@
+name: Build and public ecli docker image in non-x86
+
+on:
+  push:
+    branches: "master"
+    paths:
+      - "VERSION"
+
+jobs:
+  aarch64_job:
+    # The host should always be Linux
+    runs-on: ubuntu-22.04
+    name: Build on ubuntu-22.04 aarch64
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: update remote
+        run: |
+          git submodule update --init --recursive --remote
```

```suggestion
          submodules: 'recursive'
      - name: update remote
        run: |
```
Here is the same effect